src/Package/Impl/ProjectSystem/Commands/SetDirectoryHereCommand.cs

```diff
@@ -30,7 +30,9 @@ internal sealed class SetDirectoryHereCommand : ICommandGroupHandler {
 
         public CommandStatusResult GetCommandStatus(IImmutableSet<IProjectTree> nodes, long commandId, bool focused, string commandText, CommandStatus progressiveStatus) {
             if (commandId == RPackageCommandId.icmdSetDirectoryHere && nodes.Count == 1) {
-                return new CommandStatusResult(true, commandText, CommandStatus.Enabled | CommandStatus.Supported);
+                var session = _interactiveWorkflowProvider.Active?.RSession;
+                bool enabled = session != null && session.IsHostRunning && !session.IsRemote;
+                return new CommandStatusResult(true, commandText, enabled ? CommandStatus.Enabled | CommandStatus.Supported : CommandStatus.Supported);
```

Comment:
That doesn't exist on that particular enum (Microsoft.VisualStudio.ProjectSystem.Designers.CommandStatus)